src/app/components/user/user.component.html

```diff
@@ -39,75 +41,107 @@ <h2>{{ user.name }} {{ user.surname }}</h2>
+            <div class="statistics">
                 <span>
-                    {{ entry.key }} {{ entry.value * 100 | number:'1.1-1' }}%
+                    💬 {{ getCommentChanceOfUser(user)*100 | number:'1.1-1' }}%
                 </span>
-            }
-        </div>
-        <div class="psychoanalysis">
-            deep analyzed feelings:
-            <div class="emotion-scores">
-                @for(entry of getUserEmotionScores(user) | keyvalue; track entry.key) {
+                @for(entry of getReactionChancesOfUser(user) | keyvalue; track entry.key) {
                     <span>
-                        @if(entry.key === 'happy') { 😁 } 
-                        @else if(entry.key === 'sad') { 😢 } 
-                        @else if(entry.key === 'stupid') { 🤦‍♂️ } 
-                        @else if(entry.key === 'boring') { 😴 }
-                        {{ entry.value | number:'1.3-3' }}
+                        {{ entry.key }} {{ entry.value * 100 | number:'1.1-1' }}%
                     </span>
                 }
             </div>
-            <div>
-                bigfive:
-                <div>
-                    <span>O={{ user.big_five.openness | number:'1.3-3' }} </span>
-                    <span>C={{ user.big_five.conscientiousness | number:'1.3-3' }} </span>
-                    <span>E={{ user.big_five.extraversion | number:'1.3-3' }} </span>
-                    <span>A={{ user.big_five.agreeableness | number:'1.3-3' }} </span>
-                    <span>N={{ user.big_five.neuroticism | number:'1.3-3' }} </span>
+            <div class="psychoanalysis">
+                deep analyzed feelings:
+                <div class="emotion-scores">
+                    @for(entry of getUserEmotionScores(user) | keyvalue; track entry.key) {
+                        <span>
+                            @if(entry.key === 'happy') { 😁 } 
+                            @else if(entry.key === 'sad') { 😢 } 
+                            @else if(entry.key === 'stupid') { 🤦‍♂️ } 
+                            @else if(entry.key === 'boring') { 😴 }
+                            {{ entry.value | number:'1.3-3' }}
+                        </span>
+                    }
                 </div>
-                plutchik:
                 <div>
-                    <span>joy_sadness={{ user.plutchik.joy_sadness | number:'1.3-3' }} </span>
-                    <span>anger_fear={{ user.plutchik.anger_fear | number:'1.3-3' }} </span>
-                    <span>trust_disgust={{ user.plutchik.trust_disgust | number:'1.3-3' }} </span>
-                    <span>surprise_anticipation={{ user.plutchik.surprise_anticipation | number:'1.3-3' }} </span>
-                </div>
-                russell:
-                <div>
-                    <span>valence={{ user.russell.valence | number:'1.3-3' }} </span>
-                    <span>arousal={{ user.russell.arousal | number:'1.3-3' }} </span>
+                    bigfive:
+                    <div>
```

Comment:
Forgot to mention that we can remove this as it duplicates the Radar, just with numbers.